vidarr/wdl.py

```diff
@@ -207,6 +239,44 @@ def read_output(output: WDL.Decl):
         'workflow': doc.source_text,
         'accessoryFiles': {
             imported.uri: imported.doc.source_text for imported in doc.imports}}
+    
+    output_meta = doc.workflow.meta.get("output_meta", {})
+
+    # Iterate over each output defined in the workflow
+    for output in doc.workflow.outputs:
+
+        # Check if the output name exists in the output metadata
+        if output.name in output_meta:
+            output_metadata = output_meta[output.name]
+
+            if isinstance(output_metadata, dict) and 'vidarr_label' in output_metadata:
+                vidarr_label = output_metadata['vidarr_label']
+
+                # Define the pattern for searching and replacing the output block
+                pattern = r"(?:workflow)([\s\S]*?)(?:output\s*{)([\s\S]*?)(?=$)"
+
+                # Search for the pattern in the workflow
+                match = re.search(pattern, workflow['workflow'], re.DOTALL)
+
+                # If the pattern is found, perform the replacement
+                if match:
+
+                    # Extracting the part of the workflow string where the output block is located
+                    output_block_start = match.start(2)
+                    output_block_end = match.end(2)
+                    output_block_text = match.group(2)
+
+                    # Construct the modified output block text
+                    modified_output_block_text = output_block_text.replace(
+                    f"File {output.name} = {output.expr}",
```

Comment:
If it's the case that we're adding an additional label to an existing Pair[File,Map[String,String]], this won't match to anything.

Reply:
Yes, this is only for the file to file-with-labels case. We add a label to an existing Pair[File,Map[String,String]] at line 124, because we didn't want to use regex at all and are only using it for file to file-with-labels because infer_type didn't work as we expected, right? Should I used regex for the existing Pair[File,Map[String,String]] case as well, since it doesn't rewrite the added label at line 124, like it does here?

Reply:
Yeah, if the vidarr_label isn't in the output block in the 'workflow' part of the json, it will just get dropped. 
There's a couple ways you could approach this, `output.type` is already `isinstance` `WDL.Type.Pair` at line 250 so you could add a branch there for that case, or you expand the regex to isolate the variable type and test that (probably more difficult)